build.sbt

```diff
@@ -57,7 +57,18 @@ lazy val commonDependencies = Seq(
 )
 
 lazy val root = (project in file("."))
-  .aggregate(`support-frontend`, `support-workers`, `support-models`, `support-config`, `support-internationalisation`, `support-services`)
+  .aggregate(
+    `support-frontend`,
+    `support-workers`,
+    `support-models`,
+    `support-config`,
+    `support-internationalisation`,
+    `support-services`,
+    `support-frontend-static`
+  )
+
+lazy val `support-frontend-static` = (project in file("support-frontend-static"))
```

Comment:
this is the new project to cloudform the actual bucket only.  It would only be deployed to PROD as there will only be one bucket for all stages.  It would need to go to prod before we deploy the branch to CODE.
This may sound convoluted, but it's still better than the standard situation where we just manually create the bucket effectively in prod, and then assume that it's there.